ticdc/manage-ticdc.md

```diff
@@ -781,3 +785,16 @@ force-replicate = true
 > **Warning:**
 >
 > For tables without a valid index, operations such as `INSERT` and `REPLACE` are not reentrant, so there is a risk of data redundancy. TiCDC guarantees that data is distributed only at least once during the replication process. Therefore, enabling this feature to replicate tables without a valid index will definitely cause data redundancy. If you do not accept data redundancy, it is recommended to add an effective index, such as adding a primary key column with the `AUTO RANDOM` attribute.
+
+## Unified Sorter
+
+Unified sorter is the sorting engine in TiCDC. This feature is enabled by default. It can mitigate OOM problems caused by the following scenarios:
+
++ The data changes subscription task in TiCDC is paused for a long time, during which a large amount of incremental data is accumulated to be replicated.
```

Comment:
```suggestion
+ The data replication task in TiCDC is paused for a long time, during which a large amount of incremental data is accumulated and need to be replicated.
```
`accumulated to be replicated` sounds unnatural to me, because it sounds purposive and implies that TiCDC or some other component actively collects the data for the purpose of replication, whereas in fact TiCDC does nothing while paused.